### DISCUSSION TITLE
[Feature] Copy/Move files between Immich and External Folders

### DISCUSSION THREAD

**Discussion by Erd86** [immich-app/immich#12786] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add a button to copy/move photos between Immich and External Folders. 

copy/move from immich --> external folder
copy/move from external folder --> immich

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#10092
